far/editor.cpp

```diff
@@ -7123,6 +7123,22 @@ TEST_CASE("radix10_formatted_width")
 		{
 			return radix10_formatted_width(Value) == static_cast<short>(std::format("{}", Value).size());
 		}));
+	REQUIRE(std::ranges::all_of(
```

Comment:
This [fails](https://github.com/FarGroup/FarManager/actions/runs/17882812760/job/50852661007#step:7:291) on x86.

Reply:
I prefer commits to represent logically separate actions.
E.g. if there's a bugfix and a pre- or post-refactoring that makes sense in the context, then of course, no point in squashing them together (each should still be compilable).

On the other hand, there's no point in preserving separate commits that "just happen", e.g. some flow of thoughts or experiments or addressing comments etc.

Reply:
Here, I have a bugfix, somewhat (unrelated) refactoring, and changelog + vbuild. I think it makes more sense to have them separate.